ctapipe/ml/tools/tests/test_process.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from ctapipe.core import run_tool
+from ctapipe.io import read_table
+
+
+def test_process_apply_energy(
+    tmp_path, energy_regressor_path, prod5_gamma_lapalma_simtel_path
+):
+    from ctapipe.tools.process import ProcessorTool
+
+    output = tmp_path / "gamma_prod5.dl2_energy.h5"
+
+    config_path = tmp_path / "config.json"
+
+    input_url = prod5_gamma_lapalma_simtel_path
+
+    # la palma alpha config
```

Comment:
Is there a reason we need alpha here?
I think it just adds complexity to the test opposed to all telescopes or a smaller subset (same below)